autoload/neomake/makers/ft/markdown.vim

```diff
@@ -1,6 +1,6 @@
 function! neomake#makers#ft#markdown#EnabledMakers() abort
     let makers = executable('mdl') ? ['mdl'] : ['markdownlint']
-    return makers + ['proselint']
+    return makers + ['writegood'] + neomake#makers#ft#text#EnabledMakers()
```

Comment:
Good catch.  Should have been both proselint and writegood.
Amended and squashed, please review again.